.github/workflows/ci.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build and upload to PyPI
+on: [push]
+
+jobs:
+  build_artifacts:
+    name: Build artifacts
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: 22.9.0-2
+          use-mamba: true
+          environment-file: environment.yml
+          activate-environment: vscode-binder
+      - shell: bash -el {0}
+        run:
+          python -m build
+      - uses: pypa/gh-action-pypi-publish@v1.5.1
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
I'm not fluent enough in GH Actions :-/ my naive question: when does this `if` evaluate as true? And can we make it so that it is only true when a GitHub release is created? There is some overlap with tags, because normally you'd create a tag as part of making a release on https://github.com/betatim/vscode-binder/releases/new.

I think the main reason for linking it to a GH release instead of "just a tag" is that people might create tags for all sorts of innocent reasons and not realise that it will lead to a release.

What do you think about this?

Reply:
I have no preference about this. Personally, I make releases and tags at the same time, but some people have different thoughts about this, and thus I stick to tags by default. 

I have pushed a change that now only evaluates to `true` if the workflow was triggered by a release.